app/services/matchers/party_name_splitter.rb

```diff
@@ -49,7 +48,7 @@ def update_two_name(split_name)
     end
 
     def clean_string(string)
-      string.gsub(',', '').squish
```

Comment:
.squish not needed? is it  detrimental using it when its not needed or does it just do nothing?

Reply:
it was moved to line 8